app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+<div class="top_login">
+<%= link_to '<', root_path, class:"back_logo" %>
+<h2>Log in</h2>
+<%= f.submit "Log in", class: 'loginBtnTop' %>
+</div>
+<div class="body_login">
+  <div class="field">
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email",
+        class: 'field_input' %>
+  </div>
+
+  <div class="field">
+    <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password",
+        class: 'field_input' %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <% end %>
+
+ <%= link_to "Forgot password?", new_password_path(resource_name), class: "forgot_pass" %>
```

Comment:
- [ ]   Starting with the login page, it's still different from the design:
  This is how it looks in your website:
  
  ![image](https://user-images.githubusercontent.com/20567503/209023949-5a747ef5-0389-4af4-b073-2e9332dfaf53.png)
  
  And this is how it looks on the original design:
  
  ![image](https://user-images.githubusercontent.com/20567503/209024212-b5c84871-7bf6-4113-b3f8-2be99ac08d6b.png)
  
  Kindly fix this as it is one of the requirements.